getting_started/scripting/c_sharp/c_sharp_features.rst

```diff
@@ -42,6 +42,25 @@ and for this reason it cannot be used with value types.
     // Only call SetFrame() if mySprite is not null
     mySprite?.SetFrame(0);
 
+**Using the generic methods**
+
+Generic methods are also provided to make this type conversion transparent.
+
+``GetNode<T>()`` casts the node before returning it. It will throw an ``InvalidCastException`` if the node cannot be casted to the desired type.
```

Comment:
... if the node cannot be *cast* to the desired type

Reply:
You just blew my mind right now, I have no idea during how many years I used this verb like that...
I guess that's a good «today I learned» 😄